### ISSUE TITLE
The assimp executeable on OSX does not not.

### ISSUE THREAD

**Issue by pjako** [donny-dont/assimp-node-service#2]
Last login: Tue Feb 12 14:46:29 on ttys001
/Users/jako/dart/webgl_game/bin/tools/assimp/osx/assimp ; exit;
MyMac:~ jako$ /Users/jako/dart/webgl_game/bin/tools/assimp/osx/assimp ; exit;
dyld: Library not loaded: /Users/adam/dart/assimp/lib/libassimp.3.dylib
  Referenced from: /Users/jako/dart/webgl_game/bin/tools/assimp/osx/assimp
  Reason: image not found
Trace/BPT trap: 5
logout

[Prozess beendet]
While executing assimp on my mac directly.
It tries to get the reference of the lib in an folder that could only exist if the user is named adam ;)

**Comment by donny-dont** [comment]
Yeah its just a checked in binary from the spectre branch of my assimp repo. If you wanted to build your own version I'd use that.

Just a heads up that this repo will probably go away once there is a capable Dart web server.
